dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java

```diff
@@ -391,6 +396,30 @@ public Response aiTopicCandidateGet(@ApiParam(hidden = true) @Parameter(hidden =
         return delegate.getAITopicCandidate(id);
     }
 
+    @GET
+    @UnitOfWork
+    @Path("/aiTopicCandidates")
+    @RolesAllowed({"curator", "admin"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "getAITopicCandidates", description = "Get the AI topic candidate version for all published tools that require AI topics to be generated", security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME),
+        responses = {
+            @io.swagger.v3.oas.annotations.responses.ApiResponse(responseCode = HttpStatus.SC_OK
+                    + "", description = GetAITopicCandidates.OK_RESPONSE, content = @Content(mediaType = MediaType.APPLICATION_JSON, array = @ArraySchema(schema = @Schema(implementation = Tool.class))), headers = @Header(name = X_TOTAL_COUNT, description = "Total count of AI topic candidates", schema = @Schema(type = "integer", format = "int64"))),
+            @io.swagger.v3.oas.annotations.responses.ApiResponse(responseCode = HttpStatus.SC_UNAUTHORIZED
+                    + "", description = GetAITopicCandidates.UNAUTHORIZED_RESPONSE, content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = Error.class))),
+        })
+    public Response getAITopicCandidates(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user") @Auth User user,
+            @Parameter(in = ParameterIn.QUERY, description = "Start index of paging. If this exceeds the current result set return an empty set. If not specified in the request, this will start at the beginning of the results.") @Min(0) @DefaultValue("0") @QueryParam("offset") Integer offset,
+            @Parameter(in = ParameterIn.QUERY, description = "Amount of records to return in a given page.") @Min(1) @DefaultValue("1000") @QueryParam("limit") Integer limit,
```

Comment:
Nit, you might want a max.

In effect, it may not matter though, because we'll run this once to knock out existing topic-less entries, then daily, it will still return a small result set, even if Dockstore had 1 million published entries. But doesn't hurt.